packages/flutter/lib/src/widgets/form.dart

```diff
@@ -414,6 +417,7 @@ class FormFieldState<T> extends State<FormField<T>> {
   @protected
   void setValue(T value) {
     _value = value;
+    _valueChanged = true;
```

Comment:
This method is called by subclass textformfield.didChangeWidget.

It means the auto validate will be enabled if the textformfield was built twice with different text editing controller.

Reply:
Right, so I think I can remove here right?